php/php.editor/test/unit/data/testfiles/formatting/alignment/issue210617.php.formatted

```diff
@@ -16,35 +16,35 @@ $var111111	 = 'test';
 $var1111111	 = 'test';
 $longVarName = 'test';
 
-$v			 = 'test';
-$va			 = 'test';
-$var		 = 'test';
-$var1		 = 'test';
-$var11		 = 'test';
-$var111		 = 'test';
-$var1111	 = 'test';
-$var11111	 = 'test';
-$var111111	 = 'test';
-$var1111111	 = 'test';
-
-$v			 = 'test';
-$va			 = 'test';
-$var		 = 'test';
-$var1		 = 'test';
-$var11		 = 'test';
-$var111		 = 'test';
-$var1111	 = 'test';
-$var11111	 = 'test';
-$var111111	 = 'test';
-
-$v			 = 'test';
-$va			 = 'test';
-$var		 = 'test';
-$var1		 = 'test';
-$var11		 = 'test';
-$var111		 = 'test';
-$var1111	 = 'test';
-$var11111	 = 'test';
+$v			= 'test';
+$va			= 'test';
+$var		= 'test';
+$var1		= 'test';
+$var11		= 'test';
+$var111		= 'test';
+$var1111	= 'test';
+$var11111	= 'test';
+$var111111	= 'test';
+$var1111111 = 'test';
```

Comment:
Tab space is 4 in this test.

![nb-php-gh-6884-test-result-tab4spaces](https://github.com/apache/netbeans/assets/738383/202bfaa4-02b1-4083-aa1e-9abe59425dd3)